composer.json

```diff
@@ -20,9 +20,9 @@
         }
     ],
     "require": {
-        "php": ">=5.4",
-        "symfony/symfony": "~2.3",
-        "twig/twig": ">=1.23.0"
+        "php": ">=5.3.2",
```

Comment:
@DonCallisto Yes, indeed. I agree about your recommendation, we should use `5.6`.